.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "arrowParens": "avoid",
+  "bracketSpacing": true,
+  "printWidth": 80,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "none"
```

Comment:
This file should match the config from `eslint-config-vazco`. [Source](https://github.com/vazco/eslint-config-vazco/blob/master/configs/default.yaml#L83-L88).